lib/wine_o_inventory_api/users/member.ex

```diff
@@ -0,0 +1,20 @@
+defmodule WineOInventoryApi.Users.Member do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "members" do
+    field :name, :string
+
+    has_one :inventory, WineOInventoryApi.Inventories.Inventory, on_delete: :delete_all
```

Comment:
based on some cursory reading it seems like this is not the suggested way of doing things https://hexdocs.pm/ecto/Ecto.Schema-macro-has_one.html#has_one/3-options and it's recommended to use `references` in the migration instead.

Reply:
I don't think you need both?

Reply:
I wasn't 100% sure which is why I left it in both places. I went through some posts on the elixir forum and some people added the option only in their migration file, while others left it in both places. My plan was to see if I run into any issues when writing tests to get a better understanding of how to exactly utilize this option.